addons/languages/icelandic/apk/build.gradle

```diff
@@ -0,0 +1,8 @@
+ext.shouldBePublished = true
+ext.override_app_id = 'com.anysoftkeyboard.languagepack.icelandic_oss'
```

Comment:
No need for that, you can remove it. We own that pack(com.anysoftkeyboard.languagepack.icelandic) but didn't migrate the code here.

Reply:
done